airbyte-integrations/connectors/source-google-ads/metadata.yaml

```diff
@@ -44,6 +44,12 @@ data:
           This release upgrades the Google Ads API to the latest version (v15), which causes changes in several schemas.
           Users should refresh the source schema and reset affected streams after upgrading to ensure uninterrupted syncs.
         upgradeDeadline: "2024-01-12"
+      4.0.0:
+        message:
+          This release updates Ad Group stream schema. Field metrics.cost_micros is not available for manager accounts.
```

Comment:
You mention that this field is not available for manager accounts, what if users are not using a manager accounts, can't they still consume it?
Could we make the ad_group stream schema dynamic according to the account type?
My concern is that the scope of this breaking change might be too broad: we're removing a field for all users while it's unavailable only for a subset of users.

Reply:
Cool! It'll remain a breaking changes for a subset of users but the scope is less broad 👍